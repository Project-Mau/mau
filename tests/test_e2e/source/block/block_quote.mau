[*quote]
----
Some of the newer ones are having trouble because they never really mastered some basic techniques, but theyâ€™re working hard and improving.
----
Orson Scott Card, _Ender's Game_ (1985)
