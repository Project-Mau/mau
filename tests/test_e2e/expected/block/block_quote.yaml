data:
  args: []
  content:
  - data:
      args: []
      classes: []
      content:
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: "Some of the newer ones are having trouble because they never\
                \ really mastered some basic techniques, but they\u2019re working\
                \ hard and improving."
          kwargs: {}
          subtype: null
          tags: []
          title: {}
          type: paragraph
      engine: null
      kwargs: {}
      preprocessor: none
      secondary_content:
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: 'Orson Scott Card, '
          - data:
              args: []
              content:
              - data:
                  args: []
                  kwargs: {}
                  subtype: null
                  tags: []
                  type: text
                  value: Ender's Game
              kwargs: {}
              subtype: null
              tags: []
              type: style
              value: underscore
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: ' (1985)'
          kwargs: {}
          subtype: null
          tags: []
          title: {}
          type: paragraph
      subtype: quote
      tags: []
      title: {}
      type: block
  kwargs: {}
  subtype: null
  tags: []
  type: document

